gulpfile.js

```diff
@@ -100,4 +101,16 @@ gulp.task('clean', () => {
 	return del('prepared/**');
 });
 
-gulp.task('default', ['prepare']);
+gulp.task('serve', () => {
+	browserSync.init({
+		ui: false,
+		notify: false,
+		port: 3000,
+		server: {
+			baseDir: '.'
+		}
+	});
+	gulp.watch('index.html').on('change', browserSync.reload);
```

Comment:
I would rather do

```js
.on('change', () => {
    browserSync.reload()
});
```

in case future versions of `browser-sync` will, for example, move to es6 classes and would require proper `this`